WebApp/src/app/calendar-overview/calendar/calendar.component.ts

```diff
@@ -65,9 +65,9 @@ export class CalendarComponent implements OnInit {
 
   fetchMonthItems(newDate: Date, oldDate?: Date): void {
     if (oldDate) {
-      const foundFetchedMonthItems =
+      const didFindFetchedMonthItems =
```

Comment:
didFindFetchedMonthItems is een variabelnaam voor een boolean, maar dat is het niet.
Zou dus de naam veranderen

Reply:
![image](https://user-images.githubusercontent.com/14930277/122794076-26b4cd80-d2bc-11eb-92c8-f41fba9dc4b9.png)
Het is wel een boolean. Het voert een functie uit die een actie uitvoert en didFindFetchedMonthItems zegt of er iets geoverwrite is. Daarom heb ik het nu wel hernoemd naar didOverwriteMonthItems.